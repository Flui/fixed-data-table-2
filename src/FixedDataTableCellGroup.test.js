import React from 'react';
import FixedDataTableCellGroup from 'FixedDataTableCellGroup';
import ShallowRenderer from 'react-test-renderer/shallow';

it('renders correctly', () => {
  const props = {
  }

  const renderer = new ShallowRenderer();
  renderer.render(<FixedDataTableCellGroup {...props}></FixedDataTableCellGroup>);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});