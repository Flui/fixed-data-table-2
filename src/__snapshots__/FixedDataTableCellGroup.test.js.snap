// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="fixedDataTableCellGroupLayout_cellGroupWrapper"
  style={
    Object {
      "height": undefined,
      "left": 0,
      "width": undefined,
    }
  }
>
  <FixedDataTableCellGroupImpl
    onColumnResize={null}
  />
</div>
`;
